src/Web/CSSOM/ElementCSSInlineStyle.purs

```diff
@@ -0,0 +1,16 @@
+module Web.CSSOM.ElementCSSInlineStyle
+  ( module Exports
+  , style
+  , fromElement
+  ) where
+
+import Effect (Effect)
+import Unsafe.Coerce (unsafeCoerce)
+import Web.CSSOM.Internal.Types (ElementCSSInlineStyle) as Exports
+import Web.CSSOM.Internal.Types (ElementCSSInlineStyle, CSSStyleDeclaration)
+import Web.DOM.Element (Element)
+
+foreign import style :: ElementCSSInlineStyle -> Effect CSSStyleDeclaration
+
+fromElement :: Element -> ElementCSSInlineStyle
```

Comment:
I think this should be `fromHTMLElement` and using `Web.HTML.HTMLElement (HTMLElement)` since the spec IDL just has:

```
HTMLElement implements ElementCSSInlineStyle;
SVGElement implements ElementCSSInlineStyle;
```

(No need to worry about the SVG case just now, there isn't a `web-svg` library).

`Element` is the parent interface so both of these, but also applies to XML documents I think, where this wouldn't apply.

Reply:
done